highcharts/src/main/java/us/ascendtech/highcharts/client/chartoptions/chart/Chart.java

```diff
@@ -257,12 +257,12 @@ public final Chart setEvents(ChartEvents events) {
 	}
 
 	@JsOverlay
-	public final double getHeight() {
+	public final Object getHeight() {
```

Comment:
This will work, but it's not neccessary, instead just define it as a @JsProperty native method
Same for all getters below

Also, the @JsProperty annot. on private fields is useless;  IMO you may ditch it and it's ll work

Setters are good as they are 👍🏼

Reply:
did it